modules/local/starfusion/detect/main.nf

```diff
@@ -5,7 +5,7 @@ process STARFUSION {
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/b0/b0a58b9d30f9c72b22135f85746e10596d568c40a7d9634b13e0a0749cacd21b/data' :
```

Comment:
We would need to update the `environment.yaml` of conda as well. Plus the modules test's snapshots.